test/language/comments/hashbang-encoded-bang.js

```diff
@@ -0,0 +1,10 @@
+#\u0021
+/*---
+esid: pending
+description: >
+    Hashbang comments should not be allowed to have encoded characters
```

Comment:
Given the description, shouldn't this be a negative test expecting a SyntaxError?